Exercises/exercise_3_join_validator.md

```diff
@@ -20,7 +20,7 @@ Follow the instructions in `README.md` to make sure your node is up to date and
 As an Axelar Network validator, your Axelar node will vote on the status of Bitcoin and Ethereum transactions. To do so, it needs to be configured with your Bitcoin and Ethereum nodes using the RPC endpoint.
 
 
-1. Have an Axelar node fully caught up and running by completing the steps in `README.md`.
+1. Have an Axelar node fully caught up and running by completing the steps in `README.md`. Ensure you have some testnet coins on your validator address.
```

Comment:
+ (see below for the exact amount you'll need in your validator address)

Reply:
The faucet will give out a set amount. I will reach out to Chris so he can change the faucet amount to give enough tokens. The amount below is a minimum amount anyways, and I think users should stake more than that in case the pool increases in the future